app/components/Instructions/Instructions.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export default function Instructions({ state, }) {
+  function buildInstructionArray(appState) {
+    const instructionList = [];
+    const {
+      pathStartingPoint,
+      pathEndingPoint,
+      startButtonPressed,
+      endButtonPressed,
+    } = appState;
+    const setGoBackText = buttonText => (
+      `Press on the "${buttonText}" button in order to go back to toggle mode`
+    );
+    const setReferencePoint = buttonText => (
+      `Press on a tile to set it as the "${buttonText}" of the path`
+    );
+    const gameInstructions = {
```

Comment:
...Texts

Reply:
did you mean `gameInstructionTexts`?

Eventually decided to let the object go due to the next comment about DRYing.
was wondering if it is the right step.

The other option was defining the texts object and then destructuring it, which raised the question of why putting it inside the object to begin with..

Please advise :/

Reply:
Yes that's what I meant.
There's no right answer. Do what you think is right or just try both.